### DISCUSSION TITLE
Create table problem

### DISCUSSION THREAD

**Discussion by juanreyeseth** [serenity-is/Serenity#7014] [Help]
Greetings, I have problems. I have created a table very well, but when I create the second one, it does not appear in "Select tables for code generation (single/multiple)", only the first one I generate appears, why?
![Capture](https://github.com/serenity-is/Serenity/assets/111751840/71bccc27-26a5-4a4f-a610-742ffca87514)

**Comment by LSNicholls** [comment]
Just a guess, did you actually run the application so that the migration code runs and creates the table?  Sergen is looking at your database, via the connection strings you supplied for the project.  If the table isn't actually in the database yet, it isn't going to be listed.

**Reply by juanreyeseth** [reply]
I followed the manual step by step https://serenity.is/docs/tutorials/movies/01-creating-movie-table and it still didn't work.

**Reply by LSNicholls** [reply]
Can you verify that the table was created in the database that sergen is looking at?  I mean, go into SQL Management Studio or some other tool and see the table?  To make sure you are looking at the *correct* database, look at the connection strings in appsettings.json and ensure you are connecting to the right server, etc.  By default this is going to be something like (localdb)\mssqllocaldb.   The instructions for doing this are actually in the tutorial, I just noticed.

**And in reviewing these instructions, I just realized something else that might be involved**.  In  your screenshot you appear to be looking for Movie in alphabetic order in the dbo schema.  Notice in the screenshot in the tutorial, the Movie table is *not* in the dbo schema, it is in the **mov** schema.   I am not sure how it got there, considering the code listing for the migration script doesn't seem to specify the schema. Meanwhile, the *text* in the tutorial says "In the SQL Object Explorer, you should see the **[dbo]**.[Movie]".

I don't know which is correct (I don't use Migrations at all!), this mismatch looks like a slight bug in the tutorial instructions, although they appear to be otherwise excellent.

If you verify using another tool that the table actually exists in the Default database that matches your connection string, and if the table was actually created in the **mov** schema, as shown in the screenshot in the instructions, try cursoring down the table list in sergen, and you should see the table.

**Comment by deveracs** [comment]
Check appsettings.json and appsettings.machine.json/appsettings.development.json with different database sources

**Reply by juanreyeseth** [reply]
development only has this:
{
   "ClamAV": {
     "Enabled": false
   },
   "DetailedErrors": true,
   "Logging": {
     "LogLevel": {
       "Serenity.Data": "Debug"
     }
   },
   "StartNodeScripts": "tsbuild:watch"
}

**Comment by burcuq** [comment]
Hello. 
First of all, whatever name you created your project with, do not forget to update your project name in the migration codes you created.
Check that your namespace is correct. We prefer {TheDB__yyyyMMdd_HHmm_{Description} format in Serenity applications to maintain migration file consistency. After creating the migration, right-click on dbo.VersionInfo in your database and check whether your database has been created there. Don't forget to run the project before generating code with Sergen.

**Comment by volkanceylan** [comment]
Your two migrations are using the same migration key. And you are not following the naming style nor warnings listed in the tutorial.
